cogs/wordfilter/wordfilter.py

```diff
@@ -489,7 +499,17 @@ async def checkWords(
                     blacklistedCmd = True
 
         try:
+            # records which words were used and how often
+            filterStats = await self.config.guild(msg.guild).get_attr(KEY_USAGE_STATS)()
+
+            for word in filteredWords:
+                timesMatched = len(re.findall(r"\b" + word + r"\b", filteredMsg))
```

Comment:
1. Add flag `re.IGNORECASE` to match the behavior of `re.sub` in `_filterWord`.
2. For code clarity, I believe using `originalMsg` is better than using `filteredMsg` in this context, even though at this line they hold identical content.

```suggestion
                timesMatched = len(re.findall(r"\b" + word + r"\b", originalMsg, flags=re.IGNORECASE))
```